cupy/core/dlpack.pxd

```diff
@@ -1,5 +1,3 @@
-# DLPACK_VERSION: 010
```

Comment:
This value is apparently not used anywhere, but should we record this information in the module (and perhaps report it in `cupy.show_config()`?